e2etests/report/template.html

```diff
@@ -41,7 +41,7 @@ <h2 class="case-got">Got</h2>
       .case img {
         width: 600px;
       }
-      .case-got {
+      .case-exp + .case-got {
```

Comment:
Actually why was this change needed @gavin-ts ?

Reply:
see: before/after got label position above. (we only want to shift it over if it is a delta scenario)